media-sound/jack/files/jack-3.1.1-dontuse-jack-cursesmodule.patch

```diff
@@ -0,0 +1,59 @@
+--- a/jack_t_curses.py	2004-12-12 05:54:36.000000000 +0100
++++ b/jack_t_curses.py	2017-02-25 14:11:23.667772374 +0100
+@@ -35,11 +35,14 @@
+ try:
+     from jack_curses import endwin, resizeterm, A_REVERSE, newwin, newpad, initscr, noecho, cbreak, echo, nocbreak
+ except ImportError:
+-    warning("jack_curses module not found, trying normal curses...")
++    # we dont need jack_cursemodule any more because its included in python 
++    # since release 2.0
++    # see http://docs.python.org/whatsnew/2.0.html
++    info("jack_curses module not found, using normal curses...")
```

Comment:
You probably don't want to print that. What's the point, if we are never installing it?